lib/config/options/index.ts

```diff
@@ -1272,6 +1272,18 @@ const options: RenovateOptions[] = [
     cli: false,
     env: false,
   },
+  {
+    name: 'matchReleaseAge',
+    description:
+      'Matches the age of the new / proposed package release derived from its release timestamp. Valid only within a `packageRules` object.',
```

Comment:
```suggestion
      'Matches the age (in days) of a proposed package update. Renovate uses the package release timestamp to decide the package age. Valid only within a `packageRules` object.',
```

The current text is hard to understand. How about something like this, where we split the info into three sentences, and use simpler words?

Also, we should explain to what level of detail you can set the `matchReleaseAge` matcher. Which of these is valid input?

- `1 hour`
- `12 hours`
- `24 hours`
- `1 day`
- `1 day and 12 hours`
- `2 days`
- `3 days`
- `1 week`
- `1 month`

I'm guessing the new feature only allows time to be set based on day-intervals?

Reply:
The [Renovate docs, scheduling syntax](https://docs.renovatebot.com/key-concepts/scheduling/#scheduling-syntax) has this warning:

> [!WARNING]
> Renovate does not support scheduled minutes or "at an exact time" granularity. Granularity must be at least one hour.
>
> Renovate uses the [@breejs/later](https://github.com/breejs/later) library to parse the text, so Renovate is limited to that library's syntax support. Read the parser documentation at [breejs.github.io/later/parsers.html#text](https://breejs.github.io/later/parsers.html#text) for more details. The _@breejs/later_ library also handles the concepts of "days", time_before", and "time_after".

Is the "Granularity must be at least one hour." thing relevant to your feature too?

Reply:
I do not think so. The library you mentioned, only pops up in two files. 

https://github.com/renovatebot/renovate/blob/main/lib/workers/repository/update/branch/schedule.ts

https://github.com/renovatebot/renovate/blob/main/lib/config/migrations/custom/schedule-migration.ts

Both are related strictly to scheduling, not matching, and so is part of the documentation that you referenced.

Reply:
Hmm... I don't know what we should do anymore. :smile:

I'll let a maintainer read the description, and let them decide how to proceed. :wink:

Reply:
I think its fine, although we need to make sure docs are consistent between matchCurrentAge and matchReleaseAge